audit/audit-template.yml

```diff
@@ -29,22 +29,23 @@ Parameters:
     AllowedPattern: "(none)|(arn:aws:iam::.*:role/.*)"
 
 Conditions:
-  UsePermissionsBoundary: !Not [ !Equals [ !Ref PermissionsBoundary, "none" ] ]
-  UseCodeSigning: !Not [ !Equals [ !Ref CodeSigningConfigArn, "none" ] ]
-  UseTestRole: !Not [ !Equals [ !Ref TestRoleArn, "none" ]]
-  IsPagerDutyEnvironment: !Or [
```

Comment:
We can probably live with it, but has something on your system completely reformatted the template? Any chance you could apply your changes without changing a load of other things in a minor way? Just makes it a bit hard to check the "real" changes!

Reply:
Ah yes, that could be it, forgot which repo we were in. It might be better to disable it until we get it running in the pipeline (which might be imminent?) otherwise this will keep happening!